assets/app/view/game/corporation.rb

```diff
@@ -345,6 +345,9 @@ def render_shares
           end
 
         num_ipo_shares = share_number_str(@corporation.num_ipo_shares - @corporation.num_ipo_reserved_shares)
+        if !num_ipo_shares.empty? && @corporation.capitalization != @game.class::CAPITALIZATION
```

Comment:
How about `!num_ipo_shares.empty? ` to `num_ipo_shares.any?` avoiding the double negative

Reply:
It's a string, so it doesn't have the method `any?`. Afaict there's no equivalent for strings, the closest would be `.size > 0`, but I don't know what's preferred.